### ISSUE TITLE
Error while trying to train bigger datatset

### ISSUE THREAD

**Issue by giovannicaprio** [experiencor/keras-yolo3#102]
I have a bug dataset, with large image, and I am getting this error while trying to train it: 

ValueError: Dimension 0 in both shapes must be equal, but are 1 and 255 

<img width="1280" alt="screen shot 2018-06-12 at 21 48 44" src="https://user-images.githubusercontent.com/5255573/41324985-2d65fedc-6e8e-11e8-9029-fbb7110931bd.png">

Any ideas?

**Comment by darjoo** [comment]
It seems to be crashing when it loads the weights. 
Did you make modifications to the model?

**Comment by giovannicaprio** [comment]
@darjoo no, I did not. I used the same code to train a smaller dataset, with images 128x128 pixels

**Comment by darjoo** [comment]
@giovannicaprio Hmm that's strange. I'm not too sure what is the cause then ^^; I have never seen that error personally before.

**Comment by darjoo** [comment]
@giovannicaprio I just saw this issue post, #20. Try to lower your batch size.
I have no idea if this will work.

**Comment by giovannicaprio** [comment]
Perfect, @darjoo ! Now it is working. 

Thank you

**Comment by mehrdadazizi72** [comment]
@giovannicaprio you only change batch size? i set it to 1 and still have error
on ubuntu 18.04 and Geforce 1070-8GB
